src/modules/workflows/utils.js

```diff
@@ -8,8 +8,11 @@ const whereBuilder = (filters) => {
       /* eslint-disable-next-line no-param-reassign */
       values = [values];
     }
-
-    if (param === 'lastUpdateTs') {
+    if (param === 'excludeId' || param === 'id') {
+      where.id = where.id || {};
+      where.id[param === 'excludeId' ? 'notIn' : 'in'] = values;
+      where.id.mode = undefined;
```

Comment:
what are we using mode for ?

Reply:
this is mainly for case in sensitive search, but we don't need it here, removed